prisma/seeds/roles.ts

```diff
@@ -7,19 +7,41 @@ export async function seedRoles() {
   await prisma.roles.createMany({
     data: [
       {
-        name: 'Admin',
-        description: 'Responsible for managing a resource'
+        id: "665fdcce-8795-4cec-8c91-0ff1c47e5284",
+        name: "Admin",
+        description: "Responsible for managing a resource",
```

Comment:
Melhorar a descrição:

`Manages the entire subway system, oversees operations, and ensures compliance with policies.`